.eslintignore

```diff
@@ -4,5 +4,5 @@ Example/android/
 Example/ios/
 screenshots/
 android/
-ios/
+apple/
```

Comment:
Could you add a screenshot of the macOS SVG images to the PR :)?

Reply:
Done.